templates/header.debian.html

```diff
@@ -17,8 +17,8 @@
 
   <pre class="text-white bg-dark">
     <code>
-  echo deb [signed-by=/usr/share/keyrings/jenkins-keyring.asc] \
-    {{ web_url }} binary/ | sudo tee \
+  echo "deb [signed-by=/usr/share/keyrings/jenkins-keyring.asc]" \
+    "{{ web_url }}" binary/ | sudo tee \
```

Comment:
Since web URLs are not expected to contain shell special characters, it seems safe enough to me to use the same format here as is used in the Jenkins install guide.

Optional, since both forms are valid.
```suggestion
    {{ web_url }} binary/ | sudo tee \
```

Reply:
I've set myself a reminder to merge this in 24 hours.

Reply:
Thanks.  I think that we should include the change so that it is consistent with the installation documentation.  If `web_url` ever includes a shell special character we'll deal with it at that time.  I believe that there are cases that even the surrounding double quotes won't solve.  Rather than be inconsistent, let's retain consistency between the two sites.

Reply:
Late, I had this already on my box but didn't force push to update and then got distracted.

The only difference I had was in the message:

~~~
Quote to prevent glob expansion

NOTE: {{ web_url }} intentionally not quoted, a URL like
      https://pkg.jenkins.io/debian -or-
      https://pkg.jenkins.io/debian-stable is expected.

Issue: https://github.com/jenkinsci/packaging/issues/458
~~~

Just FYI @MarkEWaite

Reply:
Thanks @hakre!